react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt

```diff
@@ -61,4 +64,8 @@ internal class PaywallFooterViewManager : SimpleViewManager<PaywallFooterView>()
         return paywallFooterView
     }
 
+    override fun setOfferingId(view: PaywallFooterView, identifier: String) {
+        view.setOfferingId(identifier)
```

Comment:
Hello, so I was actually looking into the invalidate() method in the purchases-android
And I might be completely wrong here..

but it seems to me that the offeringId doesn't actually change inside the composeView even when you invalidate.
I forked off of both branch atm, so I can change the font to my liking from the RN side. 
But the only way for us to update the offeringId, fontProvider inside the composable view was to have a `val offeringId = mutableStateOf<String?>(null)`
and have a remember method to listen to the mutableState change.

Reply:
https://github.com/Jjastiny/purchases-android/pull/1

@vegaro I was doing something similar with the fontProvider so that react-native simpleviewmanager can change the font of the paywall and the paywall footers.
As you may know the react native bridge may create the view and edit the view after it's creation. When the props get set later in the lifecycle.

Reply:
Yes that makes sense. Thanks for pointing that out.

Reply:
@vegaro np, thx for the great product!